restapi/src/main/java/org/cristalise/restapi/ItemOutcomeAttachment.java

```diff
@@ -27,9 +27,17 @@
 import javax.ws.rs.Path;

```

Comment:
Restapi test is missing. Workflow with 2 activities: first to save a jpeg, second to save a pdf to test MIME type handlign as well.

Reply:
Can the same activity save two different mime types?

Reply:
hm, I just had a look, it is a kind of undefined. Currently there is a `AttachmentMimeTypes` Activity property, which contains the comma separated list of mine types.

Reply:
yes, the mime type can be different for each activity/outcome version